### ISSUE TITLE
Braintree_CreditCard not Found

### ISSUE THREAD

**Issue by lawrenberg** [braintree/braintree_php#44]
I am getting the following error when I try to delete a card.  I am using the current version of the SDK 
```
Request error {"message":"Class 'Braintree_CreditCard' not found","line":143,"file":"/home/lawrenberg/PhpstormProjects/Delivery-API-V2/laravel/vendor/braintree/braintree_php/lib/Braintree/Customer.php","trace":[{"function":"handleShutdown","class":"Illuminate\\Exception\\Handler","type":"->","args":[]}],"human_id":"952503"}
```

**Comment by matthewpatterson** [comment]
Hey @lawrenberg, are you still having this issue? We had a known issue with the Composer autoloader that should be fixed in [version 2.40.0](https://packagist.org/packages/braintree/braintree_php#2.40.0).

**Comment by raymondberg** [comment]
I'm going to close this issue now, but feel free to reopen if the issue persists!
